content/de/blog/hacktoberfest-2021/index.md

```diff
@@ -0,0 +1,121 @@
+{
+    "title": "Hacktoberfest 2021: Macht mit!",
+    "type": "blog",
+    "date": "2021-09-19T11:00:0+02:00",
+    "description": "Wir halten 2021 auch wieder eine Hacktoberfest-Aktion ab. Mach' bei Datenanfragen.de mit und gewinne einen Webcam-Slider, Sticker und vielleicht ein T-Shirt!",
+    "featured_image": "hacktoberfest-2021",
+    "tags": [ "hacktoberfest", "digitalocean", "github", "open source", "pull request"],
+    "authors": [ "malte", "baltpeter" ],
+    "notices": "Titelfoto angepasst nach: „[maple leaves photo](https://unsplash.com/photos/Pu-en3ew8wY)“ von [Fey Marin](https://unsplash.com/@feymarin) ([Unsplash-Lizenz](https://unsplash.com/license))"
+}
+
+
+~~O'zapft is!~~ Der Oktober und damit auch das Hacktoberfest stehen wieder vor der Tür. Auch dieses Jahr wollen wir als Datenanfragen.de e.V. wieder am Hacktoberfest teilnehmen und freuen uns auf Eure Einreichungen für unser Projekt! Wie auch schon in den letzten Jahres gibt es auch wieder etwas zu gewinnen.
```

Comment:
```suggestion
~~O'zapft is!~~ Der Oktober und damit auch das Hacktoberfest stehen wieder vor der Tür. Auch dieses Jahr wollen wir als Datenanfragen.de e. V. wieder am Hacktoberfest teilnehmen und freuen uns auf Eure Einreichungen für unser Projekt! Wie auch schon in den letzten Jahres gibt es auch wieder etwas zu gewinnen.
```

or

```suggestion
~~O'zapft is!~~ Der Oktober und damit auch das Hacktoberfest stehen wieder vor der Tür. Auch dieses Jahr wollen wir als Datenanfragen.de e.&nspb;V. wieder am Hacktoberfest teilnehmen und freuen uns auf Eure Einreichungen für unser Projekt! Wie auch schon in den letzten Jahres gibt es auch wieder etwas zu gewinnen.
```

Technically, it should actually be this but we rarely do it properly. :D 

```suggestion
~~O'zapft is!~~ Der Oktober und damit auch das Hacktoberfest stehen wieder vor der Tür. Auch dieses Jahr wollen wir als Datenanfragen.de e.&thinsp;V. wieder am Hacktoberfest teilnehmen und freuen uns auf Eure Einreichungen für unser Projekt! Wie auch schon in den letzten Jahres gibt es auch wieder etwas zu gewinnen.
```